app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+end
```

Comment:
- [ ] Make sure that you implement a method that updates the likes counter for a post.
```ruby
def update_likes_counter
  post.increment!(:likes_counter)
end
```